auth/go.mod

```diff
@@ -4,8 +4,8 @@ go 1.13
 
 require (
 	github.com/asaskevich/govalidator v0.0.0-20200108200545-475eaeb16496
-	github.com/dgrijalva/jwt-go v3.2.0+incompatible // indirect
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
```

Comment:
What's the `incompatible` referring to?

Reply:
https://github.com/dgrijalva/jwt-go/issues/294 Looks like it's a `go mod` thing, where `jwt-go` doesn't fully support modules in v3, but will do in v4. Looks like it's in preview for now, but we can upgrade if/when they release, although they mention will be some breaking changes